docs/contents/tutorials/01_getting_started.rst

```diff
@@ -103,7 +113,7 @@ So let's create a ``clicks-receiver`` subscription:
 
 .. code-block:: bash
 
-    curl -d '{"name": "clicks-receiver", "endpoint": "http://requestb.in/1isy54g1", "supportTeam": "my-team"}' -H "Content-Type: application/json" http://192.168.59.103:8080/topics/com.example.events.clicks/subscriptions
+    curl -d '{"name": "clicks-receiver", "endpoint": "http://requestb.in/1isy54g1", "supportTeam": "my-team"}' -H "Content-Type: application/json" http://:8090/topics/com.example.events.clicks/subscriptions
```

Comment:
After requesting for subscription creation I got a message saying that ``supportTeam`` field is not recognized. Are these docker images up to date?